config/application.rb

```diff
@@ -28,6 +28,15 @@ class Application < Rails::Application
       # Application configuration should go into files in config/initializers
       # -- all .rb files in that directory are automatically loaded.
 
+      # Setup redis as the cache_store.
+      # This is required for sidekiq, which uses redis to queue jobs.
+      config.cache_store = begin
+        redis_config = Rails.application.config_for(:redis).symbolize_keys
+        fail 'Redis configuration is required.' unless redis_config.present?
```

Comment:
We now need to add a redis.yml for our CF environments, to deploy/ibm-cloud/config/redis.yml